scripts/remove_zone.py

```diff
@@ -34,6 +34,15 @@ def remove_zone(zone: str):
             if zone in zone_short_name:
                 del zone_short_name[zone]
 
+    geo_json_path = ROOT_PATH / "web/geo/world.geojson"
```

Comment:
Does this work?
```suggestion
    geo_json_path = ROOT_PATH + "/web/geo/world.geojson"
```

Reply:
nvm it does :D

Reply:
Yeah https://docs.python.org/3/library/pathlib.html is pretty cool. ;)